pydo/database/database_factory.py

```diff
@@ -0,0 +1,9 @@
+from .database import Database
+from .sqlite import SqliteDatabase
+
+
+class DatabaseFactory:
+    @staticmethod
+    def database(db_type, db_name, db_path) -> Database:
+        if db_type == "sqlite":
```

Comment:
Should have an `else` and throw an exception. You could also use an enum instead of a string for `db_type`